doc/source/cookbook.rst

```diff
@@ -1354,3 +1354,37 @@ of the data values:
        'weight': [100, 140, 180],
        'sex': ['Male', 'Female']})
    df
+
+Common Excel Operations
+-----------------------
+
+Fill Handle
+***********
+
+Create a series of numbers following a set pattern in a certain set of cells.
+
+.. ipython:: python
+
+    df = pd.DataFrame({'AAA': [1]*8, 'BBB': list(range(0,8))}); df
```

Comment:
I prefer to have examples that look somehow real. I think it makes easier for users to understand what's going on.

See for example: https://github.com/pandas-dev/pandas/blob/master/pandas/core/generic.py#L789

Also, make sure that the code in the examples follow PEP-8 (there are missing spaces around `*` and after the comma in range.